### ISSUE TITLE
 [Very Minor] No Icon in Window List

### ISSUE THREAD

**Issue by FibroJedi** [pulsar-edit/pulsar#179]
### Thanks in advance for your bug report!

- [X] Have you reproduced issue in [safe mode](https://pulsar-edit.dev/docs/atom-archive/faq/sections/what-does-safe-mode-do.html)?
- [X] Have you used the [debugging guide](https://pulsar-edit.dev/docs/atom-archive/hacking-atom/#debugging) to try to resolve the issue?
- [X] Have you checked our [FAQs](https://pulsar-edit.dev/docs/launch-manual/sections/faq/) to make sure your question isn't answered there?
- [X] Does your issue already [exist](https://github.com/issues?q=is%3Aissue+user%3Apulsar-edit+is%3Aopen)?
- [X] Have you checked you are on the latest release of Pulsar?

### What happened?

When Pulsar is installed and run, no icon appears in the window list in the panel, nor in the menu.

I tried making a custom launcher and specified the pulsar.png icon. While this gave it an icon on the desktop (and when _that_ was added to the menu there was an icon) it still doesn't appear in the Window list.

As I said, very minor but is probably some "polish" needed before the official release.

Thanks for all your hard work, I'm glad to be running Pulsar so I really appreciate it.

### Which OS does this happen on?

🐧 Debian based (Linux Mint, Ubuntu, etc.)

### OS details

Linux Mint 1 (Cinnamon 5.4.12)

### Which CPU architecture are you running this on?

64-bit(x86_64)

### What steps are needed to reproduce this?

1. Install from latest .deb
2. Run the app

**Comment by Daeraxa** [comment]
So it appears it is an issue with the .deb packaging from electron-builder.

I'm on Mint 20.3 with Cinnamon 5.2.7 and with the appimage I do get an icon in grouped window list (the atom nightly logo until it is all replaced with new branding) but with the .deb I too have a missing icon.

**Comment by FibroJedi** [comment]
Okay. It also doesn't show in the normal (non-grouped) window list, so just confirming it's not applet-related really.

**Comment by johanneswilm** [comment]
I don't see the icon here either (Ubuntu 22.10). The 1024x1024 icon is somehow put here: `/usr/share/icons/hicolor/0x0/apps/pulsar.png`.

**Comment by johanneswilm** [comment]
On Ubuntu, I was able to fix this for the time being by editing the file `/usr/share/applications/pulsar.desktop` as sudo and replacing the line:

```
Icon=pulsar
```

with

```
Icon=/usr/share/icons/hicolor/0x0/apps/pulsar.png
```

**Comment by FibroJedi** [comment]
Yep that worked for me. Before that the Pulsar app in /applications/ just had the generic system icon
![pulsar-no-icon](https://user-images.githubusercontent.com/69349920/205626056-6f9b1f14-72c9-4ec4-b1e2-6dbd09504fdb.png)

Thanks for this workaround!

**Comment by Spiker985** [comment]
Interesting. From what I remember, the short form path is preferred because Ubuntu itself (and other forks) is supposed to check the global icon folders and match the filename to them, rather than the app owner specifying a specific folder

We'll need to look into it all the same, but still, slightly confusing

**Comment by johanneswilm** [comment]
I'm wondering whether "atom" is some sort of recognized name so that icon themes can override the icon and it knows where to find the icon, whereas the same is not true for "pulsar" so therefore it shows nothing unless the full path is specified?

**Comment by johanneswilm** [comment]
An alternative fix: Leave the desktop file as it is. But copy the icon like this:

```
sudo cp /usr/share/icons/hicolor/0x0/apps/pulsar.png /usr/share/pixmaps/
```

**Comment by tognee** [comment]
Same issue is present on the .rpm pachage for Fedora Linux.  
Doing:  
```
sudo ln -s /opt/Pulsar/resources/pulsar.png /usr/share/pixmaps/pulsar.png
```
makes the icon appear
